database/factories/CaseStudyFactory.php

```diff
@@ -22,7 +22,7 @@ public function definition(): array
             "photo" => sprintf("%s/%s", "factory", "case_study.jpg"),
             "published" => $this->faker->boolean,
             "company" => $this->faker->company,
-            "slug" => $this->faker->slug,
+            "slug" => $this->translations($locales, $this->faker->slug),
```

Comment:
Same here

Reply:
fixed, came from previous pull request (before your changes in code)